docs/synctl-create-test.md

```diff
@@ -21,7 +21,7 @@ synctl create test [options]
     --retries <int>                     retry times, value is from [0, 2]
     --retry-interval <int>              retry interval, range is [1, 10]
     --timeout <num>ms|s|m               set timeout, accept <number>(ms|s|m)
-    --custom-properties <string>        An object with name/value pairs to provide additional information of the Synthetic test
+    --custom-property <string>          An object with name/value pairs to provide additional information of the Synthetic test
```

Comment:
I think we don't change this property, test configuration use `customProperties` as the name of custom property, if we change it, it will cause another Incompatible issue.
<img width="734" alt="image" src="https://github.com/instana/synthetic-synctl/assets/25629629/8ab84f33-5222-4302-afdf-2272c07ff38b">